src/main.ts

```diff
@@ -76,23 +89,40 @@ async function getApps(): Promise<App[]> {
   } catch (e) {
     core.error(e);
   }
+  core.info(
+    `github.context.payload.pull_request?.head?: ${JSON.stringify(
+      github.context.payload.pull_request?.head
+    )}`
+  );
   return (responseJson.items as App[]).filter(app => {
-    // TODO filter apps to only ones where they point to paths that have changed in this repo
-    return app.spec.source.repoURL.includes(
-      `${github.context.repo.owner}/${github.context.repo.repo}`
+    core.info(JSON.stringify(app));
+    core.info(`app.spec.source.targetRevision: ${app.spec.source.targetRevision}`);
+    return (
+      app.spec.source.repoURL.includes(
+        `${github.context.repo.owner}/${github.context.repo.repo}`
+      ) &&
+      (app.spec.source.targetRevision === 'master' ||
```

Comment:
super nit: is it possible for the Action to introspect which branches it's enabled on, rather than hard-coding `master` in both the source and Action usage?

Reply:
I'll look into this in a followup PR